pkgs/development/tools/analysis/graudit/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, groff
+, which
+, zip
+}:
+
+stdenv.mkDerivation rec {
+  pname = "graudit";
+  version = "3.4";
+
+  src = fetchFromGitHub {
+    owner = "wireghoul";
+    repo = pname;
```

```suggestion
    repo = "graudit";
```

https://github.com/NixOS/nixpkgs/issues/277994